webapp/components/user_settings/user_settings_general.jsx

```diff
@@ -936,6 +963,99 @@ class UserSettingsGeneralTab extends React.Component {
+                extraInfo = (
+                    <span>
+                        <FormattedMessage
+                            id='user.settings.general.field_handled_externally'
+                            defaultMessage='This field is handled through your login provider. If you want to change it, you need to do so though your login provider.'
+                        />
+                    </span>
+                );
+            } else {
+                let positionLabel = (
+                    <FormattedMessage
+                        id='user.settings.general.position'
+                        defaultMessage='Position'
+                    />
+                );
+                if (Utils.isMobile()) {
+                    positionLabel = '';
+                }
+
+                inputs.push(
+                    <div
+                        key='positionSetting'
+                        className='form-group'
+                    >
+                        <label className='col-sm-5 control-label'>{positionLabel}</label>
+                        <div className='col-sm-7'>
+                            <input
+                                className='form-control'
+                                type='text'
+                                onChange={this.updatePosition}
+                                value={this.state.position}
+                                maxLength={Constants.MAX_POSITION_LENGTH}
+                                autoCapitalize='off'
+                            />
+                        </div>
+                    </div>
+                );
+
+                extraInfo = (
+                    <span>
+                        <FormattedMessage
+                            id='user.settings.general.positionExtra'
+                            defaultMessage='Tell your teammates what you do.'
+                        />
+                    </span>
+                );
+
+                submit = this.submitPosition;
+            }
+
+            positionSection = (
+                <SettingItemMax
+                    title={formatMessage(holders.position)}
+                    inputs={inputs}
+                    submit={submit}
+                    server_error={serverError}
+                    client_error={clientError}
+                    updateSection={(e) => {
+                        this.updateSection('');
+                        e.preventDefault();
+                    }}
+                    extraInfo={extraInfo}
+                />
+            );
+        } else {
+            let describe = '';
+            if (user.position) {
+                describe = user.position;
+            } else {
+                describe = (
+                    <FormattedMessage
+                        id='user.settings.general.emptyPosition'
+                        defaultMessage="Click 'Edit' to add your position"
```

Comment:
I don't think this string says much, is it related to your **job** position? maybe discuss this with a PM?